plugin/src/test/java/org/opensearch/ml/rest/RestMLUpdateConnectorActionTests.java

```diff
@@ -137,6 +138,15 @@ public void testPrepareRequestFeatureDisabled() throws Exception {
         restMLUpdateConnectorAction.handleRequest(request, channel, client);
     }
 
+    public void testPrepareRequestUpdateDisabled() throws Exception {
+        exceptionRule.expect(IllegalStateException.class);
+        exceptionRule.expectMessage(UPDATE_CONNECTOR_DISABLED_ERR_MSG);
+
+        when(mlFeatureEnabledSetting.isUpdateConnectorEnabled()).thenReturn(false);
+        RestRequest request = getRestRequest();
+        restMLUpdateConnectorAction.handleRequest(request, channel, client);
+    }
+
```

Comment:
This feature flag was only needed because the pending security review? Since the recent PRs will address the security review concerns, we can remove this feature flag? But I do think we can add these update API to the feature flag of "remote inference" with a default "true" setting.

Reply:
Sure I will change it to true.